libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -185,64 +172,61 @@ irods::error start(irods::default_re_ctx& _u,const std::string& _instance_name)
 }
 
 irods::error stop(irods::default_re_ctx& _u,const std::string& _instance_name) {
-
     std::lock_guard<std::mutex> lock(audit_plugin_mutex);
 
-    json_t* obj = json_object();
-    if( !obj ) {
-        return ERROR(SYS_MALLOC_ERR, "json_object() failed");
+    irods::error ret = get_re_configs( _instance_name );
+    if( !ret.ok() ) {
+        irods::log(PASS(ret));
     }
 
+    nlohmann::json json_obj;
+
     struct timeval tv;
     gettimeofday(&tv, NULL);
     unsigned long time_ms = tv.tv_sec * 1000 + tv.tv_usec / 1000;
     std::stringstream time_str; time_str << time_ms;
-    json_object_set(obj, "time_stamp", json_string(time_str.str().c_str()));
+    json_obj["time_stamp"] = time_str.str(); 
 
     char host_name[MAX_NAME_LEN];
     gethostname( host_name, MAX_NAME_LEN );
-    json_object_set(obj, "hostname", json_string(host_name));
+    json_obj["hostname"] = std::string(host_name);
 
     pid_t pid = getpid();
     std::stringstream pid_str; pid_str << pid;
-    json_object_set(obj, "pid", json_string(pid_str.str().c_str()));
+    json_obj["pid"] = pid_str.str();
+
+    json_obj["action"] = "STOP";
 
-    json_object_set(obj, "action", json_string("END"));
```

Comment:
Should 198 be like this?
```c++
    json_obj["action"] = "END";
```
Just noticed that the value changed from "END" to "STOP".